src/resources/dispatch/DispatchList.tsx

```diff
@@ -78,7 +78,15 @@ export default function DispatchList(props: DatagridProps): React.ReactElement {
     : 'simple-dispatch-list'
 
   return (
-    <List hasCreate storeKey={storeKey} filters={filters}>
+    <List
+      hasCreate
+      storeKey={storeKey}
+      filters={filters}
+      filterDefaultValues={
+        process.env.MOCK
```

Comment:
yeah, if we want to see items that have not been dispatched, we should be checking for `false`